docs/Data-Model.md

```diff
@@ -31,30 +31,40 @@ For example, a JSON document describing software configuration may look like thi
 
 ```json
 {
-  "repositories": [
-    {
-      "alias": "YaST:Head",
-      "name": "YaST:Head",
-      "type": "rpm-md",
-      "url": "http://download.opensuse.org/repositories/YaST:/Head/openSUSE_12.3/",
-      "enabled": true,
-      "autorefresh": true,
-      "gpgcheck": true,
-      "priority": 99
+  "repositories": {
+    "_attributes": {
+      "repository_system": "zypp"
     },
-    ...
-  ],
-  "packages": [
-    {
-      "name": "kernel-default",
-      "version": "3.0.101",
-      "release": "0.35.1",
-      "arch": "x86_64",
-      "vendor": "SUSE LINUX Products GmbH, Nuernberg, Germany",
-      "checksum": "2a3d5b29179daa1e65e391d0a0c1442d"
+    "_elements": [
+      {
+        "alias": "YaST:Head",
+        "name": "YaST:Head",
+        "type": "rpm-md",
+        "url": "http://download.opensuse.org/repositories/YaST:/Head/openSUSE_12.3/",
+        "enabled": true,
+        "autorefresh": true,
+        "gpgcheck": true,
+        "priority": 99
+      },
+      ...
+    ]
+  },
+  "packages": {
+    "_attributes": {
+      "package_system": "rpm"
     },
-    ...
-  ],
+    "_elements": [
+      {
+        "name": "kernel-default",
+        "version": "3.0.101",
+        "release": "0.35.1",
+        "arch": "x86_64",
+        "vendor": "SUSE LINUX Products GmbH, Nuernberg, Germany",
+        "checksum": "2a3d5b29179daa1e65e391d0a0c1442d"
+      },
+      ...
+    ]
+  },
```

Comment:
It would be good to explain the meaning of the special attributes `_attributes` and `_elements` and what consequences they have for the data model.